cocos/2d/CCNode.cpp

```diff
@@ -1235,7 +1246,7 @@ void Node::visit(Renderer* renderer, const Mat4 &parentTransform, uint32_t paren
         {
             auto node = _children.at(i);
 
-            if (node && node->_localZOrder < 0)
+            if (node && node->getLocalZOrder() < 0)
```

Comment:
`_loacalZOrder` is defined as `unsigned long long` in this PR, then `node->getLocalZOrder() < 0` will be false for ever.